src/toast/todmap/pointing.py

```diff
@@ -33,7 +33,7 @@ class OpPointingHpix(Operator):
     the total response is:
 
     .. math::
-        d = cal \\left[\\frac{(1+eps)}{2} I + \\frac{(1-eps)}{2} \\left[Q \\cos{4(a+w)} + U \\sin{4(a+w)}\\right]\\right]
+        d = cal \\left[\\frac{(1+eps)}{2} I + \\frac{(1-eps)}{2} \\left[Q \\cos{2a+4w} + U \\sin{2a+4w}\\right]\\right]
```

Comment:
@giuspugl Is this definition of the pointing weights consistent with what Davide used for the LB simulations?